include/deal.II/lac/filtered_matrix.h

```diff
@@ -187,6 +187,9 @@ template <class VectorType> class FilteredMatrixBlock;
  * not need to serialize calls to @p vmult or @p residual .
  *
  * @author Wolfgang Bangerth 2001, Luca Heltai 2006, Guido Kanschat 2007, 2008
+ *
+ * @deprecated Use LinearOperator instead. See the documentation of
+ * constrained_linear_operator.
```

Comment:
Write this as `constrained_linear_operator()` so that doxygen automatically links to it. (Assuming that this function lives in the global namespace.)